utils.groovy

```diff
@@ -959,4 +959,54 @@ def build_and_push_image(params = [:]) {
     return manifest_digest
 }
 
+def get_arch_image_digest(from) {
+    def node_image_digest = shwrapCapture(""" skopeo inspect --raw docker://${from} """)
+    def digest_arch_map = shwrapCapture(""" echo '${node_image_digest}' | jq -r '.manifests[] | \
+                                      select(.platform.os == \"linux\") | \"\\(.platform.architecture)=\\(.digest)\"'
+                        """.stripIndent())
+    def arch_digests = [:]
+    digest_arch_map.split("\\n").each { line ->
+        def (arch, digest) = line.tokenize("=")
+        arch_digests[arch] = digest
+    }
+    return arch_digests
```

Comment:
Minor/optional: I think we could simplify it to this:

```suggestion
    return readJson(text: shwrapCapture("""
        skopeo inspect --raw docker://${from} | \
            jq -r '.manifests | map({(.platform.architecture): .digest}) | add'
    """))
```

Reply:
A  nice this map here! Good to know!